seonshine_backend/controllers/commonController.js

```diff
@@ -230,3 +231,74 @@ const getAdminDashboardSummary = async () => {
 
   return response;
 };
+
+export const getCurrentProfile = async (req, res) => {
+  try {
+    const { user_id, role_id } = req.user;
+    const numberRoleId = Number(role_id);
+
+    const userAttributes = [
+      'user_id',
+      'role_id',
+      'username',
+      'email',
+      'phone_number',
+      'user_status',
+    ];
+
+    const profileAttributes =
+      numberRoleId === UserRole.restaurant
+        ? ['address', 'profile_picture_url']
+        : ['birth_date', 'address', 'profile_picture_url'];
```

Comment:
both roles have the same `address` and `profile_picture_url`. Please define default profileAttributes and check is Restaurant role to push `birth_date`